### PR TITLE
Styling and small refactors for visual purposes

### PR DESCRIPTION
**Pull request by stevkim**

### CODE REVIEW

File: src/components/account/AccountSetting.jsx [hunk 1]
```diff
@@ -34,25 +34,32 @@ const AccountSetting = () => {
   };
```
Comment (bzw24) [inline src/components/account/AccountSetting.jsx]: I split the updateUserData axios calls in the utils file, not sure why the axios call was added back.

**Review by bzw24 — APPROVED** [review]
Good with all the styling consolidation, thanks for driving this effort with us on our meeting. See note on Accounts Setting. Thought we wanted to separate axios calls into the utils folder.

**Review by justcord10 — APPROVED** [review]
Looks good, I really like the burgundy accents added into the page, all the other changes tie the whole site together!

**Review by justcord10 — CHANGES_REQUESTED** [review]
See Bruces comment/review

**Review by bzw24 — APPROVED** [review]

**Review by bzw24 — APPROVED** [review]

**Review by justcord10 — APPROVED** [review]
Good changes! thanks!

**Pull request merged**